src/test/java/edu/ucla/library/bucketeer/verticles/ClearCacheIT.java

```diff
@@ -91,7 +90,7 @@ public class ClearCacheIT {
     /**
      * The URL for the IIIF image server.
      */
-    private String myIiifURL;
+    private String myIIIfURL;
```

Comment:
I think this should have been the first one: myIiifURL. Could also be myImageServerURL though since the downcasing of the acronym can be weird.